src/routes/safe/components/Transactions/TxList/BatchExecute.tsx

```diff
@@ -129,11 +131,11 @@ export const BatchExecute = React.memo((): ReactElement | null => {
   const dispatch = useDispatch<Dispatch>()
   const { address: safeAddress, currentVersion } = useSelector(currentSafe)
   const account = useSelector(userAccountSelector)
-  const safeInstance = getGnosisSafeInstanceAt(safeAddress, currentVersion)
```

Comment:
@usame-algan you shouldn't do expensive operations like this in the render function. It potentially called this function dozens of times.